.travis.yml

```diff
@@ -0,0 +1,28 @@
+before_script:
+- sudo apt-get install nodejs
```

Comment:
I guess there isn't any way around this, but isn't it generally bad practice to have `sudo`s in scripts? @stumash

Reply:
confirmed